failFast

```diff
@@ -1 +1 @@
-true
+false
```

Comment:
temporarily

Reply:
Removed the temporary failFast when the job passed on ci.jenkins.io